decadog/src/command/sprint.rs

```diff
@@ -234,21 +238,38 @@ fn finish_sprint(settings: &Settings) -> Result<(), Error> {
         };
 
         println!("{}", &issue);
+        // This bool keeps track of whether an issue was planned or not. Issues are considered
+        // planned if they belong to the current milestone at time of closing the sprint. If an
+        // issue is added to the milestone at the end of the sprint, then it is considered as out
+        // of sprint.
+        issue_is_planned = true;
 
-        // If no milestone, ask to assign to open milestone
+        // If no milestone, ask to assign to open milestone and if applicable mark as not planned
         // If answer is no, ignore this issue
         if issue.milestone.is_none() {
             if Confirmation::new("Assign to milestone?").interact()? {
                 client.assign_issue_to_milestone(&issue, Some(&open_milestone))?;
+                issue_is_planned = false;
             } else {
                 continue;
             }
         };
 
         let zenhub_issue = client.get_zenhub_issue(&repository, &issue)?;
-        if zenhub_issue.estimate.is_none() {
-            let estimate = select_estimate.interact()?;
-            client.set_estimate(&repository, &issue, estimate.value)?;
+        let estimate = zenhub_issue.estimate;
+        let estimate_value: u32;
+        if let Some(existing_estimate) = estimate {
+            estimate_value = existing_estimate.value;
+        } else {
+            let new_estimate = select_estimate.interact()?;
+            estimate_value = new_estimate.value;
+            client.set_estimate(&repository, &issue, new_estimate.value)?;
+        }
```

Comment:
I would write getting the estimate value as:

```rust
let estimate_value = if let Some(existing_estimate) = estimate {
    existing_estimate.value
} else {
    let new_estimate = select_estimate.interact()?;
    client.set_estimate(&repository, &issue, new_estimate.value)?;
    new_estimate.value
}
```

This avoids the the bare `let estimate_value: u32;` and shadowing the variable later on.

I would also remove `let estimate = zenhub_issue.estimate;` as you only use the `estimate` variable in one place.